### PR TITLE
Shuffle Big Red disk locations

### PR DESCRIPTION
**Pull request by RipGrayson**
## About The Pull Request
Apparently Big Red disks were either too easy or too hard, this PR adjusts them to the following locations
Red has been moved from Marshal's to Atmos. 
Blue is a room farther north in ETA. 
Green has been moved from Viro into Admin.

## Why It's Good For The Game

People have been complaining for a bit about Big Red disk locations, this should make things better balanced.

## Changelog
:cl:
balance: Shuffled Big Red nuclear generator computers.
balance: Moved a single platinum miner.
/:cl:

### CODE REVIEW

**Review by TiviPlus — CHANGES_REQUESTED** [review]
Atomize the tank out pls

Also on the tank you only really need the assembled and base parts, no? since its just a prop

**Comment by RipGrayson** [comment]
> Also on the tank you only really need the assembled and base parts, no? since its just a prop
The point of all other decorations is to allow mappers to make modular tanks with various other parts on them. Say I need a beaten up tank with broken treads and a shredded minigun and ruined TOW launcher, you can't do that with the complete tank prop.

File: code/game/objects/structures/prop.dm [hunk 1]
```diff
@@ -608,3 +608,388 @@
...
+	bound_width = 64
+	resistance_flags = RESIST_ALL
+	layer = ABOVE_MOB_LAYER
+
+/obj/structure/prop/vehicle/tank/north
+	icon = 'icons/Marine/tank_propns.dmi'
+	icon_state = "tank_complete"
+
+/obj/structure/prop/vehicle/tank/north/base
+	name = "Tank"
```
Comment (XSlayer300) [inline code/game/objects/structures/prop.dm:629]:
```suggestion
	name = "decommissioned TAV - Rhino"
```

File: code/game/objects/structures/prop.dm [hunk 2]
```diff
@@ -608,3 +608,388 @@
...
+
+/obj/structure/prop/vehicle/tank/north/altnorth
+	icon_state = "tank_complete_alt"
+
+/obj/structure/prop/vehicle/tank/east
+	icon = 'icons/Marine/tank_propew.dmi'
+	icon_state = "tank_complete"
+
+/obj/structure/prop/vehicle/tank/east/base
+	name = "Tank"
```
Comment (XSlayer300) [inline code/game/objects/structures/prop.dm:641]:
```suggestion
	name = "decommissioned TAV - Rhino"
```

File: code/game/objects/structures/prop.dm [hunk 3]
```diff
@@ -608,3 +608,388 @@
...
+	icon_state = "artillerymod_0"
+
+/obj/structure/prop/vehicle/tank/north/armor/snowplow
+	icon_state = "snowplow_0"
+
+/obj/structure/prop/vehicle/tank/north/armor/snowplowone
+	icon_state = "snowplow_1"
+
+/obj/structure/prop/vehicle/apc
+	name = "APC"
```
Comment (XSlayer300) [inline code/game/objects/structures/prop.dm:904]:
```suggestion
	name = "decommissioned TAV - Athena"
```

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**